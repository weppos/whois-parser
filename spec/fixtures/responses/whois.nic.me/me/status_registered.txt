Domain Name: GOOGLE.ME
Domain ID: D108500000000011599-AGRS
WHOIS Server:
Referral URL: http://www.markmonitor.com
Updated Date: 2016-05-12T09:26:23Z
Creation Date: 2008-06-13T17:17:40Z
Registry Expiry Date: 2017-06-13T17:17:40Z
Sponsoring Registrar: MarkMonitor Inc.
Sponsoring Registrar IANA ID: 292
Domain Status: clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Domain Status: clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited
Domain Status: serverDeleteProhibited https://icann.org/epp#serverDeleteProhibited
Domain Status: serverTransferProhibited https://icann.org/epp#serverTransferProhibited
Domain Status: serverUpdateProhibited https://icann.org/epp#serverUpdateProhibited
Registrant ID: a74791c75837f786
Registrant Name: DNS Admin
Registrant Organization: Google Inc.
Registrant Street: 1600 Amphitheatre Parkway
Registrant City: Mountain View
Registrant State/Province: CA
Registrant Postal Code: 94043
Registrant Country: US
Registrant Phone: +1.6502530000
Registrant Phone Ext:
Registrant Fax: +1.6502530001
Registrant Fax Ext:
Registrant Email: dns-admin@google.com
Admin ID: a74791c75837f786
Admin Name: DNS Admin
Admin Organization: Google Inc.
Admin Street: 1600 Amphitheatre Parkway
Admin City: Mountain View
Admin State/Province: CA
Admin Postal Code: 94043
Admin Country: US
Admin Phone: +1.6502530000
Admin Phone Ext:
Admin Fax: +1.6502530001
Admin Fax Ext:
Admin Email: dns-admin@google.com
Tech ID: a74791c75837f786
Tech Name: DNS Admin
Tech Organization: Google Inc.
Tech Street: 1600 Amphitheatre Parkway
Tech City: Mountain View
Tech State/Province: CA
Tech Postal Code: 94043
Tech Country: US
Tech Phone: +1.6502530000
Tech Phone Ext:
Tech Fax: +1.6502530001
Tech Fax Ext:
Tech Email: dns-admin@google.com
Name Server: NS1.GOOGLE.COM
Name Server: NS2.GOOGLE.COM
Name Server: NS4.GOOGLE.COM
Name Server: NS3.GOOGLE.COM
DNSSEC: unsigned
>>> Last update of WHOIS database: 2016-08-02T23:40:13Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

Access to WHOIS information is provided to assist persons in determining the contents of a domain name registration record in the registry database. The data in this record is provided by The Registry Operator for informational purposes only, and accuracy is not guaranteed.  This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to(a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (b) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or Afilias except as reasonably necessary to register domain names or modify existing registrations. All rights reserved. Registry Operator reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy.
