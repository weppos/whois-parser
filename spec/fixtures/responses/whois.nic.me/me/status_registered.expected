#disclaimer
  %s == "Access to WHOIS information is provided to assist persons in determining the contents of a domain name registration record in the registry database. The data in this record is provided by The Registry Operator for informational purposes only, and accuracy is not guaranteed.  This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to(a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (b) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or Afilias except as reasonably necessary to register domain names or modify existing registrations. All rights reserved. Registry Operator reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy."


#domain
  %s == "google.me"

#domain_id
  %s == "D108500000000011599-AGRS"


#status
  %s == ["clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited", "clientTransferProhibited https://icann.org/epp#clientTransferProhibited", "clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited", "serverDeleteProhibited https://icann.org/epp#serverDeleteProhibited", "serverTransferProhibited https://icann.org/epp#serverTransferProhibited", "serverUpdateProhibited https://icann.org/epp#serverUpdateProhibited"]

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2008-06-13 17:17:40 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2016-05-12 09:26:23 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2017-06-13 17:17:40 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "292"
  %s.name         == "MarkMonitor Inc."
  %s.organization == nil
  %s.url          == nil

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type         == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id           == "a74791c75837f786"
  %s[0].name         == "DNS Admin"
  %s[0].organization == "Google Inc."
  %s[0].address      == "1600 Amphitheatre Parkway"
  %s[0].city         == "Mountain View"
  %s[0].zip          == "94043"
  %s[0].state        == "CA"
  %s[0].country_code == "US"
  %s[0].phone        == "+1.6502530000"
  %s[0].fax          == "+1.6502530001"
  %s[0].email        == "dns-admin@google.com"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type         == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id           == "a74791c75837f786"
  %s[0].name         == "DNS Admin"
  %s[0].organization == "Google Inc."
  %s[0].address      == "1600 Amphitheatre Parkway"
  %s[0].city         == "Mountain View"
  %s[0].zip          == "94043"
  %s[0].state        == "CA"
  %s[0].country_code == "US"
  %s[0].phone        == "+1.6502530000"
  %s[0].fax          == "+1.6502530001"
  %s[0].email        == "dns-admin@google.com"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type         == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id           == "a74791c75837f786"
  %s[0].name         == "DNS Admin"
  %s[0].organization == "Google Inc."
  %s[0].address      == "1600 Amphitheatre Parkway"
  %s[0].city         == "Mountain View"
  %s[0].zip          == "94043"
  %s[0].state        == "CA"
  %s[0].country_code == "US"
  %s[0].phone        == "+1.6502530000"
  %s[0].fax          == "+1.6502530001"
  %s[0].email        == "dns-admin@google.com"


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.google.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.google.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns4.google.com"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns3.google.com"
